apps/extension/src/ui/apps/dashboard/routes/Settings/Accounts/AccountsLogoStack.tsx

```diff
@@ -0,0 +1,72 @@
+import { AccountJsonAny } from "@core/domains/accounts/types"
+import { WithTooltip } from "@talisman/components/Tooltip"
+import { classNames } from "@talismn/util"
+import { AccountIcon } from "@ui/domains/Account/AccountIcon"
+import useAccounts from "@ui/hooks/useAccounts"
+import { useMemo } from "react"
+import styled from "styled-components"
+
+type Props = { addresses?: string[]; className?: string; max?: number }
+
+export const AccountsLogoStack = ({ addresses, className, max = 4 }: Props) => {
+  const accounts = useAccounts()
+
+  const filteredAccounts = useMemo(
+    () => accounts.filter((account) => addresses?.includes(account.address)),
+    [accounts, addresses]
+  )
+  const { visibleAccounts, moreAccounts } = useMemo(
+    () => ({
+      visibleAccounts: filteredAccounts?.slice(0, max) ?? [],
+      moreAccounts: filteredAccounts?.slice(max) ?? [],
+    }),
+    [filteredAccounts, max]
+  )
+
+  return (
+    <div className={classNames("logo-stack pl-[0.25em]", className)}>
```

Comment:
I think this is copy/pasted from portfolio ? 
Shouldn't need this logo-stack class, it was only meant as a way to override this child, when we were using styled components

```suggestion
    <div className={classNames("pl-[0.25em]", className)}>
```